src/main/java/org/commcare/cases/model/Case.java

```diff
@@ -118,6 +122,22 @@ public void setExternalId(String id) {
         data.put(EXTERNAL_ID_KEY, id);
     }
 
+    public String getCategory() {
+        return (String)data.get(CATEGORY_KEY);
```

Comment:
I think this is where you want to be aliasing the values. 

IE: The "Category" index should be a lookup of the key `category` if that key exists in the set, or a lookup of the key `patient_type` if it doesn't.

Alternatively you could do that aliasing in the `getMetaFields` block. I'm not sure, honestly, which one would result in less confusing behavior for someone who's not expecting it.